lib/charms/opensearch/v0/constants_charm.py

```diff
@@ -38,3 +38,7 @@
 AdminUserInitProgress = "Configuring admin user..."
 HorizontalScaleUpSuggest = "Horizontal scale up advised: {} shards unassigned."
 WaitingForOtherUnitServiceOps = "Waiting for other units to complete the ops on their service."
+
+# Relation Interfaces
+ClientRelationName = "opensearch-client"
+PeerRelationName = "opensearch-peers"
```

Comment:
nit: do we need the "Name" suffix there or would `PeerRelation` be enough?

Reply:
In my view, the `Name` suffix is necessary because we have `ops` relation objects and our `provider` objects that both represent the _relation_ more accurately - I wanted to make it obvious that this only relates directly to the name.

Reply:
**suggestion**: just mentioning, but according to the [pep](https://peps.python.org/pep-0008/#constants), constants in python are usually identified by all upper case and separated with underscore. With camelcase, the names rather look like classes otherwise. 

But I understand this may need quite a refactoring as I see above a number of constants in camel cases. Just mentioning. As a suggestion, up to you whether address this

Reply:
yes, sure. Up to you, as mentioned in the comment it is a "suggestion" (non blocking comments left to the author) and I leave it up to you decide how to proceed on this